src/Message.js

```diff
@@ -34,6 +34,27 @@ const styles = {
 
 export default class Message extends React.Component {
 
+  constructor(props) {
+    super(props)
+    this.state = {
+      mentionedBG:false
+    }
+  }
+
+  componentDidMount() { 
+    console.log("mention , curr",this.props.mentionedMsgId,this.props.currentMessage._id )
+    if(this.props.mentionedMsgId) {
+      this.setState({mentionedBG :true})
+    } else{
+      this.setState({mentionedBG :false})
+    }
+
+    setTimeout(() => {
+      this.setState({mentionedBG :false})
+    }, 1000);
+   }
+
```

Comment:
do bar same code kyon daala h , same is in bubble file

Reply:
hata dia ab, expermental tha kuch